sources/osgShadow/ShadowMap.js

```diff
@@ -63,11 +65,15 @@ define( [
         apply: function ( node ) {
             var st = node.getStateSet();
             // check that and other things ?
+            // TODO: should check whole hierarchy in fact...
             if ( st ) {
                 var blend = st.getAttribute( 'BlendFunc' );
                 if ( blend !== undefined && blend.getSource() !== BlendFunc.DISABLE ) {
```

Comment:
It's either try to detect it, however imperfectly like here or it's user work to specify shadowcast node mask by himself on all it's alpha blended geoms